playwright-e2e/lib/component/dsm/filters/sections/customize-view.ts

```diff
@@ -67,10 +77,11 @@ export class CustomizeView {
 
   /* XPaths */
   private get openButtonXPath(): string {
-    return (
-      "//div[text()[normalize-space()='Customize View'] and " +
-      "button[.//*[local-name()='svg' and @data-icon='server']/*[local-name()='path']]]/button"
-    );
+    return `//*[text()[normalize-space()="Customize View"]]/button`;
+  }
+
+  private async isPanelOpen(): Promise<boolean> {
+    return await this.page.locator('.btn-group').count() >= 1;
```

Comment:
This function helps to determine whether Customize View panel is open or closed.